src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -0,0 +1,253 @@
+                : [BylineCss.imageLtr, BylineCss.Image]
+            }
+          >
+            <img css={BylineCss.imageSource} src={image} alt="" />
+          </li>
+        )}
+        <li>
+          {authorTopicUrl ? (
+            <React.Fragment>
+              <VisuallyHiddenText>{`${authorTranslated}, `}</VisuallyHiddenText>
+              <a
+                css={[BylineCss.link, BylineCss.authorLink]}
+                href={authorTopicUrl}
+              >
+                <strong
+                  className="byline__link-text"
+                  css={[
+                    BylineCss.author,
+                    getSansBold(service),
+                    getBodyCopy(script),
+                  ]}
+                >
+                  {author}
+                </strong>
+                {isRtl ? (
+                  <LeftChevron css={BylineCss.authorChevron} />
+                ) : (
+                  <RightChevron css={BylineCss.authorChevron} />
+                )}
+              </a>
+            </React.Fragment>
+          ) : (
+            <span role="text">
+              <VisuallyHiddenText>{`${authorTranslated}, `}</VisuallyHiddenText>
+              <strong
+                css={[
+                  BylineCss.author,
+                  BylineCss.authorLink,
+                  getSansBold(service),
+                  getBodyCopy(script),
+                ]}
+              >
+                {author}
+              </strong>
+            </span>
+          )}
+        </li>
+        <li>
+          <span role="text">
+            <VisuallyHiddenText>{`${jobRoleTranslated}, `} </VisuallyHiddenText>
+            <strong
+              css={[
+                BylineCss.jobRole,
+                getSansBold(service),
+                getBrevier(script),
+              ]}
+            >
+              {jobRole}
+            </strong>
+          </span>
+        </li>
+        {twitterLink ? (
+          <li>
+            <a
+              css={[BylineCss.link, BylineCss.twitterLink]}
+              href={twitterLink}
+              aria-labelledby="byline-twitter-link"
+            >
+              <span role="text" id="byline-twitter-link">
+                <VisuallyHiddenText lang="en-GB">
+                  {`Twitter, `}
+                </VisuallyHiddenText>
+                <span
+                  className="byline__link-text"
+                  css={[
+                    BylineCss.twitterText,
+                    getSansBold(service),
+                    getBrevier(script),
+                  ]}
+                >{`@${twitterText}`}</span>
+                {isRtl ? (
+                  <LeftChevron css={BylineCss.twitterChevron} />
+                ) : (
+                  <RightChevron css={BylineCss.twitterChevron} />
+                )}
+              </span>
+            </a>
+          </li>
+        ) : null}
+        {location ? (
+          <li>
+            <span css={BylineCss.location}>
```

Comment:
Would of thought role text might be needed here so this content is announced with one gesture with AT to avoid text splitting?